vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1042,8 +1058,11 @@ def generate_draft_token_ids(
             if drafter_output is None or len(drafter_output) == 0:
                 draft_token_ids.append([])
             else:
+                #assert len(drafter_output) <= max_gen_len
+                #draft_token_ids[i] = drafter_output
                 draft_token_ids.append(drafter_output.tolist())
```

Comment:
We could keep this as a list of ndarrays. Later could even update the drafter to take an ndarray as input and have it write the tokens into it, so we can build a single 1-dim array with offsets.